tests/smartswitch/common/device_utils_dpu.py

```diff
@@ -37,29 +37,31 @@ def check_smartswitch_and_dark_mode(duthosts,
     """
 
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    output = duthost.os_version
+    os_version = float(re.search(r'([1-9]{1}\d*)(\.\d{0,2})', output).group())
```

Comment:
this pattern is not working with our internal images, and it is also not working with master branch. Could  you use the existing skip_release method? https://github.com/sonic-net/sonic-mgmt/blob/9c124855d7d70a121deda59978e710cd06cf2ff2/tests/common/utilities.py#L72

Reply:
Actually, smartswitch case is little different, it can be run on os versions greater than 202405.
The check of DPUS field in platform.json is enough as well to check that.

Reply:
So removed this check altogether